src/healthRecord/healthRecord.interface.ts

```diff
@@ -1,6 +1,7 @@
 import { ApiProperty } from "@nestjs/swagger"
-import { IsArray, IsNumber, IsObject, IsString } from "class-validator"
+import { IsArray, IsNumber, IsObject, IsOptional, IsString } from "class-validator"
 import { UploadProfileRequest } from "src/user/dto/user.dto"
+import { IsNull } from "typeorm"
```

Comment:
Is this used anywhere? dont forget to remove unused import.

Reply:
UploadProfileRequest is not used by anyone too. Should I remove it too?

Reply:
Removed